SuperBuild.cmake

```diff
@@ -1,16 +1,30 @@
 include(ExternalProject)
 
-set(Autoscoper_DEPENDENCIES
+# set(Autoscoper_DEPENDENCIES
+#   GLEW
+#   TIFF
+#   )
+
+# foreach(dependency IN LISTS Autoscoper_DEPENDENCIES)
+#   message(STATUS "SuperBuild - Adding ${dependency}")
+#   include(${CMAKE_CURRENT_SOURCE_DIR}/Superbuild/External_${dependency}.cmake)
+# endforeach()
+set(proj ${SUPERBUILD_TOPLEVEL_PROJECT})
+
+# Project dependencies
+set(${proj}_DEPENDS
   GLEW
   TIFF
+)
+
+ExternalProject_Include_Dependencies(${proj}
+  PROJECT_VAR proj
```

Comment:
```suggestion
  PROJECT_VAR proj
  DEPENDS_VAR Autoscoper_DEPENDENCIES
```

Reply:
By explicitly describing the name of the `DEPENDS` variable, you would not have to rename it.

See https://cmake-artichoke.readthedocs.io/en/latest/ExternalProjectDependency.html#function:ExternalProject_Include_Dependencies